week_1/String/longest_common_prefix.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {string[]} strs
+ * @return {string}
+ */
+const longestCommonPrefix = (words) => {
```

Comment:
Can we add some tests? As it stands, I don't have a great way of confirming if the behavior here works or not

Reply:
I added the test file, but it's so long that it's difficult to sift through. I think the file needs to be split up. Do you think that having a separate file for each test would be extra/confusing?

Reply:
Usually you'd have a test file for each logic file. 

So you'd see something like
- `string/longest_common_prefix.js`
- `string/longest_common_prefix.test.js`

both sitting next to each other. It's not uncommon to see them in a test directory like you have here, too.